src/SimpleCalculator/Calculator.cs

```diff
@@ -2,15 +2,19 @@ namespace SimpleCalculator;
 
 public class Calculator
 {
-    public static void Main()
+    public static void Main(string[]args)
     {
+        Addition add = new Addition();
+        Subtraction subtract = new Subtraction();
+        Division divide = new Division();
+        Multiplication multiply = new Multiplication();
         try
         {
             do
             {
                 int userOption;
                 System.Console.WriteLine("Welcome, dear!");
```

Comment:
nitpick: please use [namespaces](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/types/namespaces) for a shorter code